scss/mixins/_viewport-logic.scss

```diff
@@ -39,29 +41,33 @@ $susy: (
 
 @each $zone-index in $grid-zones {
   .grid-zone.grid_#{$zone-index} {
-    @include span($zone-index of $current-columns no-gutter);
+    @if $zone-index < $current-columns {
+      @include span($zone-index of $current-columns nest);
 
-    @for $blok-index from 1 through $current-columns {
-      .grid-blok.grid_#{$blok-index} {
-        @include span($blok-index of $zone-index no-gutter);
+      @for $blok-index from 1 through $current-columns {
+        @if $blok-index < $zone-index  {
+          .grid-blok.grid_#{$blok-index} {
+            @include span($blok-index of $zone-index);
 
-        > .grid-element {
-          @include span($blok-index of $blok-index);
+            > .grid-element {
+              @include span($blok-index of $blok-index $outer-padding);
+            }
+          }
         }
       }
     }
   }
 
-  @if $zone-index > $current-columns {
+  @if $zone-index >= $current-columns {
```

Comment:
What does this do btw? Why are `.grid-blok`s different when the `$zone-index` exceed the `$current-columns`